docs/user/customization.md

```diff
@@ -0,0 +1,67 @@
+
+## Platform Customization
+
+While the default cluster size may be sufficient for some, many will need to make alterations. This can include increasing the number of machines in the control plane, changing the type of the virtual machines that will be used (e.g. AWS instances), or adjusting the CIDR range used for the Kubernetes service network. This level of customization is exposed via the installer's `install-config.yaml`. The install-config can be accessed by running `openshift-install create install-config`. This file can then be modified as needed before running a later target.
+
+The `install-config.yaml` generated by the installer will not have all of the available fields populated, so they will need to be manually added if they are needed. The full list of available fields can be found in the [Go Docs][godocs]. Documentation for each of the supported platforms can be found in their platform-specific section:
+
+- [AWS][aws-customization]
+
+[aws-customization]: aws/customization.md
+[godocs]: https://godoc.org/github.com/openshift/installer/pkg/types#InstallConfig
+
+## Kubernetes Customization (unmanaged)
+
+In addition to customizing OpenShift and aspects of the underlying platform, the installer allows arbitrary modification to the Kubernetes objects that are injected into the cluster. Note that there is currently no validation on the modifications that are made, so it is possible that the changes will result in a non-functioning cluster. The Kubernetes manifests can be viewed and modified using the `manifests` and `manifest-templates` targets.
+
+The `manifests` target will render the manifest templates and output the result into the asset directory. Perhaps the most common use for this target is to include additional manifests in the initial installation. These manifests could be added after the installation as a "Day 2" operation, but there may be cases where they are necessary beforehand.
+
+The `manifest-templates` target will output the unrendered manifest templates into the asset directory. This allows modification to the templates before they have been rendered, which may be useful to users who wish to reuse the templates between cluster deployments.
+
+## OS Customization (unmanaged)
+
+In rare circumstances, certain modifications to the bootstrap and other machines may be necessary. The installer provides the "ignition-configs" target, which allows arbitrary modification to the [Ignition Configs][ignition] used to boot these machines. Note that there is currently no validation on the modifications that are made, so it is possible that the changes will result in a non-functioning cluster.
```

Comment:
[QE wants][1] an additional sentence here along the lines of "any customization via Ignition configurations is unsupported", although personally I'm not clear on what that line suggests that your current "Note that there is currently no validation... will result in a non-functioning cluster" does not.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1666165#c20

Reply:
If we do not express that "any customization via Ignition configurations is unsupported", user/customer would customize their cluster via Ignition configurations following their mind, and treat that is something we have to support, but the truth is thing is changing over time, some customization via Ignition files do not take effect any more, just like what we discussed in BZ#1666165, that seem like open an avail box for our supported matrix.

Reply:
> ... but the truth is thing is changing over time...

We already document that [here][1] with "the content of the generated files may change".  And Ignition customizations will continue to have effects, it's just that those effects may get stomped on by other cluster components.

[1]: https://github.com/openshift/installer/blame/v0.10.1/docs/user/versioning.md#L7

Reply:
Supported vs unsupported is not a distinction that needs to be made in a _project_. This sort of distinction should be made in the _product_ instead. I'll follow up with the documentation group to make sure that it is clear in the product documentation.